tests/checkup_test.go

```diff
@@ -223,6 +223,16 @@ func newKubeVirtDPDKCheckerRole() *rbacv1.Role {
 				Resources: []string{"virtualmachineinstances"},
 				Verbs:     []string{"create", "get", "delete"},
 			},
+			{
```

Comment:
In the commit message it says that the RoleBinding is also changed, while in the code it does not.

Reply:
DONE